motionplan/constraint.go

```diff
@@ -270,6 +270,14 @@ func createAllCollisionConstraints(
 		constraintMap[defaultObstacleConstraintDesc] = obstacleConstraint
 	}
 
+	if len(boundingRegions) > 0 {
+		// create constraint to keep moving geometries within the defined bounding regions
+		unionOfRobotGeom := movingRobotGeometries
+		unionOfRobotGeom = append(unionOfRobotGeom, staticRobotGeometries...)
```

Comment:
We can just use moving robot geoms; static geoms do not need to be (and should not be) included in bounding regions